integration-tests/smoke/mercury_test.go

```diff
@@ -0,0 +1,42 @@
+package smoke
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/rs/zerolog/log"
+	"github.com/smartcontractkit/chainlink/integration-tests/testsetups"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMercurySmoke(t *testing.T) {
+	const mercuryFeedId = "ETH-USD-Optimism-Goerli-1"
+	_, isExistingTestEnv, testNetwork, chainlinkNodes, mercuryServerInternalUrl,
+		evmClient, mockServerClient, mercuryServerClient := testsetups.SetupMercuryEnv(t)
+
+	if isExistingTestEnv {
+		log.Info().Msg("Use existing Mercury test env")
+	} else {
+		log.Info().Msg("Creating new Mercury test env..")
+		nodesWithoutBootstrap := chainlinkNodes[1:]
+		ocrConfig := testsetups.BuildMercuryOCR2Config(t, nodesWithoutBootstrap)
+		verifier, _ := testsetups.SetupMercuryContracts(t, evmClient, mercuryFeedId, ocrConfig)
+		testsetups.SetupMercuryNodeJobs(t, chainlinkNodes, mockServerClient, verifier.Address(),
+			mercuryFeedId, mercuryServerInternalUrl, testNetwork.ChainID, 0)
+		// Set OCR2 config in the contract
+		verifier.SetConfig(ocrConfig)
+		// Wait for the DON to start generating reports
+		d := 160 * time.Second
```

Comment:
Can we use retry here?
```
gom := gomega.NewGomegaWithT(t)
	gom.Eventually(func(g gomega.Gomega) { ... }
```

Reply:
@skudasov good idea, let me do it in a separate PR